_codux/boards/team-cards.board.css

```diff
@@ -0,0 +1,78 @@
+.teamCards {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    font-family: Arial, sans-serif;
+    min-height: 800px;
+    max-height: 700px;
+
+    @media (width < 1024px) {
+        min-height: 500px;
+    }
+    @media (width < 768px) {
+        min-height: 400px;
+    }
+    @media (width < 480px) {
+        flex-direction: column;
+        min-height: 300px;
+    }
+}
+
+.teamCardsCard {
+    outline: 1px solid rgba(255, 255, 255, 0.5);
+    box-shadow: 3px 3px 10px rgba(0, 0, 0, 0.1);
+    position: relative;
+    overflow: hidden;
+}
+
+.teamCardsCardImage {
+    object-fit: cover;
+    height: 100%;
+    width: 100%;
+}
```

Comment:
why as image and not background-image? This way you don't have to make the content in absolute position.